src/main/java/core/basesyntax/GetDraw.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface GetDraw {
+    abstract void draw();

```

Comment:
In Java, all methods in an interface are implicitly `abstract`, so you don't need to include the `abstract` keyword. You can simply write `void draw();`.